boards/FSM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -143,7 +147,69 @@ void App_SetPeriodicSignals_AcceleratorPedal(const struct FsmWorld *world)
 
 void App_SetPeriodicSignals_MotorShutdownFaults(const struct FsmWorld *world)
 {
-    struct FsmCanTxInterface *can_tx = App_FsmWorld_GetCanTx(world);
+    struct FsmCanTxInterface *can_tx      = App_FsmWorld_GetCanTx(world);
+    struct FsmCanRxInterface *can_rx      = App_FsmWorld_GetCanRx(world);
+    static uint8_t            error_count = 0;
```

Comment:
```suggestion
    static uint8_t            error_count = 0;
```

can be more specific with the error count name

Reply:
renamed to `torque_plausibility_err_cnt`